src/main/java/biz/paluch/spinach/output/JobOutput.java

```diff
@@ -35,13 +39,24 @@ public void set(ByteBuffer bytes) {
             return;
         }
 
-        body = codec.decodeValue(bytes);
+        if (body == null) {
+            counters = new HashMap<String, Long>();
```

Comment:
Not sure what you mean here (sorry, don't write java very often) - declare `counters = new HashMap<String, Long>()` at class level declaration or something else?